android/app/src/main/kotlin/coop/polypoly/polypod/Authentication.kt

```diff
@@ -0,0 +1,77 @@
+package coop.polypoly.polypod
+
+import android.content.Context
+import android.widget.Toast
+import androidx.biometric.BiometricManager
+import androidx.biometric.BiometricPrompt
+import androidx.core.content.ContextCompat
+import androidx.fragment.app.FragmentActivity
+
+class Authentication {
+    companion object {
+        private val desiredLockScreenType =
+            BiometricManager.Authenticators.BIOMETRIC_WEAK or
+                BiometricManager.Authenticators.DEVICE_CREDENTIAL
+
+        fun shouldShowBiometricsPrompt(context: Context): Boolean {
+            return biometricsAvailable(context) &&
+                Preferences.getBiometricCheck(context) &&
+                !Preferences.getBiometricEnabled(context) &&
+                !Preferences.isFirstRun(context)
```

Comment:
This is one of the two changes to the flow: The user should _not_ be asked to set up biometrics on the very first launch.l

In theory we could simplify the Onboarding activity now, since the use case for including it with the other onboarding slides is gone. But I left that logic intact since I see a bit of a chance that we'll go back to it :D But yeah, me leaving unused code behind - not typical!